ui/lib/kmip/addon/components/operation-field-display.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @module OperationFieldDisplay
+ * OperationFieldDisplay components are used on KMIP role show pages to display the allowed operations on that model
+ *
+ * @example
+ * ```js
+ * <OperationFieldDisplay @model={{model}} />
+ * ```
+ *
+ * @param model {DS.Model} - model is the KMIP role model that needs to display its allowed operations
+ *
+ */
+import Component from '@ember/component';
+import layout from '../templates/components/operation-field-display';
+
+export default Component.extend({
+  layout,
+  tagName: '',
+  model: null,
+
+  trueOrFalseString(model, field, returnTrue, returnFalse) {
```

Comment:
I think naming here is a little confusing. Perhaps `trueString` and `falseString` instead? Or comments about the params?

Reply:
Updated in a8a646a !